python/dynamic_graph/sot/torque_control/create_entities_utils.py

```diff
@@ -266,6 +269,7 @@ def create_balance_controller(robot, conf, motor_params, dt, robot_name='robot')
     plug(robot.estimator_ft.contactWrenchRightSole, ctrl.wrench_right_foot);
     plug(robot.estimator_ft.contactWrenchLeftSole,  ctrl.wrench_left_foot);
     plug(ctrl.tau_des,                              robot.torque_ctrl.jointsTorquesDesired);
+    plug(ctrl.dv_des,                               robot.torque_ctrl.jointsAccelerationsDesired);
```

Comment:
Actually dv_des contains also the base accelerations, which should not appear in jointAccelerationsDesired!

Reply:
Reading the code below I see that actually you take only the last nbJoints values of the signal jointsAccelerationsDesired, so the behavior of the system is correct. In this, the problem is just the name of the signal jointsAccelerationsDesired that is not correct. I suggest to call it dv_des, as the output signal of the balance controller.

Reply:
Since the integration scheme is to be applied only to the joint accelerations and not to the base acceleration, I think your earlier comment might be more suitable. I'll change the size to NJ in the code and use vector selector to plug the required parts from ctrl.dv_des.